crates/driver/openapi.yml

```diff
@@ -331,6 +331,12 @@ components:
             Information about tokens used in the auction.
         deadline:
           $ref: "#/components/schemas/DateTime"
+        surplusCapturingJitOrderOwners:
+          type: array
+          items:
+            $ref: "#/components/schemas/Address"
+          description: |
+            List of surplus capturing JIT-order owners.
```

Comment:
This sentence can literally be autogenerated from the name and type. Maybe we should give some more context/documentation here? Imaging you are a solver consuming this API documentation and want to understand what this field means. How about something like:

List of addresses on whose behalf the solver is allowed to create "just-in-time" orders for and whose surplus will count towards the objective value of their solution (unlike other orders that were created by the solver).